contracts/JBveBanny.sol

```diff
@@ -344,23 +345,14 @@ contract JBveBanny is veERC721, Ownable, ReentrancyGuard, JBOperatable {
         _requirePermission(_ownerOf, projectId, JBStakingOperations.EXTEND_LOCK);
 
       // Calculate the new unlock date
-      uint256 _newEndDate = ((block.timestamp + _data.updatedDuration));
-      if (_newEndDate < _lock.end) revert JBErrors.INVALID_LOCK_DURATION();
+      uint256 _newEndDate = (block.timestamp + _data.updatedDuration);
+      if (_newEndDate < _lock.end) revert INVALID_LOCK_EXTENSION();
 
-      // Set the new end date
-      _lock.end = _newEndDate;
+      // TODO: Add back in the changing tokenId, temporarily removed to improve gas usage
```

Comment:
can you specify what exactly was added here to get some clarity

Reply:
I reintroduced [this](https://discord.com/channels/775859454780244028/961007548553637970/966339972833624166) bug/foot gun (idk what to call it) since I didn't want to run the logic to burn and then create a new locker because of the high gas usage.

I might have a new idea on a gas efficient way we can fix this. I'll explain my idea in the discord so we can get some feedback from the others as well.

Reply:
got it yeah we can focus on this once we wrap up the ve nft but yeah the packing approach is interesting

Reply:
yep got it